src/components/graph/Graph.jsx

```diff
@@ -300,6 +305,9 @@ export default class Graph extends React.Component {
 
     if (!this.state.config.freezeAllDragEvents) {
       zoomObject.on("zoom", this._zoomed);
+      if (this.onDragChange) {
+        zoomObject.on("end", this._zoomedEnd);
```

Comment:
What would be the use case for this?

Reply:
the use case is when we move the graph we need to handle the final position for save it in bdd and after reload we send an init position to the graph (i've not finish to code this part)